README.md

```diff
@@ -25,111 +26,115 @@ KubePlus takes an application Helm chart and wraps it in a Kubernetes API (CRD).
+   `minikube start`
 
-   ``$ minikube start ``
+   or
 
-2) Download KubePlus plugins and set up the PATH
-```
-  wget "https://github.com/cloud-ark/kubeplus/blob/master/kubeplus-kubectl-plugins.tar.gz?raw=true"
-  mv kubeplus-kubectl-plugins.tar.gz\?raw\=true kubeplus-kubectl-plugins.tar.gz
-  gunzip kubeplus-kubectl-plugins.tar.gz
-  tar -xvf kubeplus-kubectl-plugins.tar
-  export KUBEPLUS_HOME=`pwd`
-  export PATH=$KUBEPLUS_HOME/plugins:$PATH
-  kubectl kubeplus commands
-```
+   `kind create cluster`
 
-3) Set the Namespace in which to deploy KubePlus
+3. Unzip KubePlus plugins and set up the PATH
 
-   ``export KUBEPLUS_NS=default``
+   ```
+   tar -zxvf kubeplus-kubectl-plugins.tar.gz
+   export KUBEPLUS_HOME=`pwd`
+   export PATH=$KUBEPLUS_HOME/plugins:$PATH
+   kubectl kubeplus commands
+   ```
+
+4. Set the Namespace in which to deploy KubePlus
+
+   `export KUBEPLUS_NS=default`
 
-4) Create provider kubeconfig using provider-kubeconfig.py
+5. Create provider kubeconfig using provider-kubeconfig.py
 
    ```
-   wget https://raw.githubusercontent.com/cloud-ark/kubeplus/master/requirements.txt
-   wget https://raw.githubusercontent.com/cloud-ark/kubeplus/master/provider-kubeconfig.py
-   wget https://raw.githubusercontent.com/cloud-ark/kubeplus/master/parse-api-server-url.sh
    python3 -m venv venv
    source venv/bin/activate
    pip3 install -r requirements.txt
    apiserver=`./parse-api-server-url.sh`
    python3 provider-kubeconfig.py -s $apiserver create $KUBEPLUS_NS
+   deactivate
    ```
 
-5) Install KubePlus Operator using the generated provider kubeconfig 
+6. Install KubePlus Operator using the generated provider kubeconfig 
 
    ```
    helm install kubeplus "https://github.com/cloud-ark/operatorcharts/blob/master/kubeplus-chart-3.0.39.tgz?raw=true" --kubeconfig=kubeplus-saas-provider.json -n $KUBEPLUS_NS
    until kubectl get pods -A | grep kubeplus | grep Running; do echo "Waiting for KubePlus to start.."; sleep 1; done
    ```
 
-6) Create Kubernetes CRD representing WordPress Helm chart.
+7. Create Kubernetes CRD representing WordPress Helm chart.
    
-   The WordPress Helm chart can be specified as a [public url](./examples/multitenancy/wordpress/wordpress-service-composition.yaml#L14) or can be [available locally](./examples/multitenancy/wordpress/wordpress-service-composition-localchart.yaml#L14).
+   *The WordPress Helm chart can be specified as a [public url](./examples/multitenancy/wordpress/wordpress-service-composition.yaml#L14) or can be [available locally](./examples/multitenancy/application-hosting/wordpress/wordpress-service-composition.yaml).*
 
    ```
-   kubectl create -f  https://raw.githubusercontent.com/cloud-ark/kubeplus/master/examples/multitenancy/wordpress/wordpress-service-composition.yaml --kubeconfig=kubeplus-saas-provider.json
+   kubectl create -f examples/multitenancy/application-hosting/wordpress/wordpress-service-composition.yaml --kubeconfig=kubeplus-saas-provider.json
```

Comment:
Some background on why we have absolute paths here.
We have gone back and forth on whether to list the relative paths or absolute paths. For relative paths, the repo has to be cloned. And given the size of the repo, we felt that it might become a blocker for someone to try the example. That was the reason we did not have the "clone the repo" as the first step. Anyone who wants to try, should be able to do so without having to first clone the repo. Of course, once the repo size is fixed cloning will not be an issue. And as a temporary solution, shallow clone also will be helpful.
What do you think?